### ISSUE TITLE
Windows app parameters not consistent for file globbing

### ISSUE THREAD

**Issue by collinpeters** [oracle/graal#3066]
**Describe the issue**
Native images produced on Windows do not correctly do any file globbing on parameters.

**Steps to reproduce the issue**
Please include both build steps as well as run steps
1. With the following code
```
public class SimpleArgs
{
    public static void main(String[] args) {
        for (String arg : args) {
            System.out.println("Arg: " + arg);
        }
    }
}
```
2. On Linux this works fully as expected. On Linux systems the shell is the one doing the file globbing
```
$ java SimpleArgs *
Arg: foo
Arg: bar
$ ./simpleargs *
Arg: foo
Arg: bar
```
3. On Windows the native-image does *not* behave the same as the JRE.
```
c:\Code>java SimpleArgs *
Arg: foo
Arg: bar
c:\Code>simpleargs *
Arg: *
```

As I understand it, on Windows the shell does not do any globbing and the Windows JRE actually has functionality replicated in it. Assuming this is true it would ideally mean that functionality is replicated in Graal native-image.

**Describe GraalVM and your environment:**
 - GraalVM version - 20.2
 - JDK major version: 11
 - OS: Linux (latest Arch), Windows 10
 - Architecture: AMD64

**More details**
Don't think there is any other relevant info. I can gather more if needed.

**Comment by mcraj017** [comment]
@collinpeters thanks for reporting the issue
we will check and get back to you

**Comment by collinpeters** [comment]
The release notes for 21.1 read:

> Improved support on Windows: applications now do file globbing on parameters.

Does that resolve this issue? Haven't had a chance to test myself.

**Comment by collinpeters** [comment]
Can confirm this is fixed in 21.1
